app.js

```diff
@@ -52,12 +53,20 @@ app.use(session({
     store : new MongoStore({mongooseConnection: mongoose.connection})
 }))
 
+// ** set body bodyParser
+// app.use(bodyParser.urlencoded({ extended: false }));
 
 
 // ** passport middleware 
 app.use(passport.initialize());
 app.use(passport.session());
 
+// ** set global variable
+app.use(function(req,res,next) {
```

Comment:
Set global variable , ther are two type of variable story user and logged in user that why need a global variable

Reply:
ok